test/unit/argument_parser/format_parse_test.cpp

```diff
@@ -921,9 +921,10 @@ TEST(parse_test, subcommand_argument_parser_success)
 
     // incorrect sub command
     {
```

Comment:
please add the issue number here to track that this was a related change.

Reply:
```suggestion
    { // see issue https://github.com/seqan/seqan3/issues/2172
```